internal/merger/merger_test.go

```diff
@@ -226,7 +226,15 @@ func TestMergeNode(t *testing.T) {
 			},
 		},
 		{
-			&pbv2.NodeResponse{},
+			&pbv2.NodeResponse{
+				Data: map[string]*pbv2.LinkedGraph{
+					"dcid1": {
+						Arcs: map[string]*pbv2.Nodes{
+							"prop1.2": {},
```

Comment:
Perhaps also add a test case where the value is `nil`?

```
Arcs: map[string]*pbv2.Nodes{
  "prop1.2": nil,
}
```

Reply:
Done